CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 The changes are relative to the previous release, unless the baseline is specified.
 
-## [Unreleased]
+## [1.2.0] - 2025-02-13
```

Comment:
This should be a new line, leaving the `## [Unreleased]` line unchanged. See the [v1.1.1 commit](https://github.com/AOMediaCodec/libavif/commit/bb24db03cd99befe09c87b602e36f24d75a980d1) for an example.

Reply:
Done. I thought it would be cleaner to put `## [Unreleased]` back in the next PR.